### ISSUE TITLE
How can I place controls outside of the swipable area?

### ISSUE THREAD

**Issue by AndrewBogdanovTSS** [surmon-china/vue-awesome-swiper#628]
@surmon-china could you add an example of the swiper with arrows outside of the swipable area and how can that be achieved with the current swiper architecture? I would like to get this look https://take.ms/YC1Zw
Right now `swiper` class get overflow-hidden by default that means that controls will cut off if a negative margin is applied. Please, help me understand how to work around this issue?

**Comment by surmon-china** [comment]
@AndrewBogdanovTSS 

1. remove `navigation option` from `swiperOptions`
2. remove `navigation slot element`
3. custom element and bind event to control the swiper

e.g.

```html
<div class="container">
  <swiper ref="swiper">
    <swiper-slide>Slide 1</swiper-slide>
  </swiper>
  <div class="navigation">
    <button @click="prev">Prev</button>
    <button @click="next">Next</button>
  </div>
</div>
```

```js
prev() {
  this.$refs.swiper.$swiper.slidePrev()
},
next() {
  this.$refs.swiper.$swiper.slideNext()
}
```

**Comment by AndrewBogdanovTSS** [comment]
@surmon-china clicking behavior of controls is not the only aspect of the functionality they provide. Using such approach how can I replicate the behavior of disabling controls when this is needed? For example if there are not enough slides to swipe?

**Comment by imfunniee** [comment]
@AndrewBogdanovTSS don't know if this is the right way to do it but you can use ``this.$refs.swiper.$swiper.progress`` to detect that. 

Basically if its less that "0.1" there aren't any slides to the left and if its greater than "0.9" there aren't any slides to the right.

You can check it on each click or you can use a watch().

**Comment by AndrewBogdanovTSS** [comment]
@imfunniee I don't fill this is a right way of doing it. @surmon-china maybe you could recommend something better?

**Comment by johnyluyte** [comment]
@AndrewBogdanovTSS https://github.com/surmon-china/vue-awesome-swiper/issues/146#issuecomment-621117460

**Comment by AndrewBogdanovTSS** [comment]
@johnyluyte yeah, I also end up with similar solution. But the problem with it is that it's not universal and always requires to pass that `customClassName` for every swiper instance which is very inconvenient and counter intuitive so I would rather call it a hack rather than something that can be accepted as a proper fix
